content/documentation/staging/configuration/health/index.adoc

```diff
@@ -0,0 +1,384 @@
+---
+title: "Health Configuration"
+date: 2019-03-06T13:55:38+02:00
+draft: false
+type: "documentation"
+weight: 5
+---
+
+:linkattrs:
+:sectlinks:
+
+= Health Configuration
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Custom Health
+:keywords: Kiali Documentation Health
+:icons: font
+:imagesdir: /images/documentation/health-configuration/
+
+
+== Health
+
+Customize resource health by overriding the default health configuration.
```

Comment:
Again, please don't assume the user knows how health works.
It requires an intro, that you will see pods / traffic and probably more signals in the future.

A global health is calculated as the max severity of all inputs/signals.

This feature allows to determine the severity of those signals.